modules/overview/pages/introduction-to-eclipse-che.adoc

```diff
@@ -55,7 +55,7 @@ Learn about the latest of {prod} and submit your blog posts to the link:https://
 
 Weekly meetings::
 
-Join us in the link:https://github.com/eclipse/che/wiki/{prod-short}-Dev-Meetings[{prod-short} community meeting] every Monday.
+Join us in the link:https://github.com/eclipse/che/wiki/{prod-id}-community-meetings[{prod-short} community meeting] every Monday.
```

Comment:
```suggestion
Join us in the link:https://github.com/eclipse/che/wiki/{prod-id-short}-community-meetings[{prod-short} community meeting] every Monday.
```

Reply:
The url should be https://github.com/eclipse-che/che/wiki/Eclipse-Che-Community-Meetings

{prod-id-short} will be represented as **che** and the url will be broken (https://github.com/eclipse-che/che/wiki/che-community-meetings)

Thats why i used {prod-id} which will be represented as **eclipse-che** and the url will work correctly (https://github.com/eclipse-che/che/wiki/eclipse-che-community-meetings)

Reply:
understood. thank you for the clarification.